### ISSUE TITLE
Refill Booking Form with Routing Form fields effortlessly.

### ISSUE THREAD

**Issue by hariombalhara** [calcom/cal.com#7292]
Hello. I have a question. Is there a way you can pre-populate the calendar form that ask name and email with the info from a routing form. So we can improve the user experience by filling just one a form. Thank you!

[Slack Message](https://calendso.slack.com/archives/C01NXDSRY9W/p1677093470875399?thread_ts=1677093470.875399&cid=C01NXDSRY9W)

**Comment by rksingh2001** [comment]
Can I take up this issue?

**Comment by rksingh2001** [comment]
Can you assign this to me @hariombalhara ?

**Comment by hariombalhara** [comment]
@rksingh2001 sure assigning you. But I would like you to share your approach as well, before doing a lot of code, as it might not be straightforward.

**Comment by rksingh2001** [comment]
Hello @hariombalhara, do we want to remove the fields in case they doesn't exist or prepopulate them?

**Comment by hariombalhara** [comment]
I think prepopulating but also marking them hidden would be a better and simple option.

**Comment by rksingh2001** [comment]
Well pre-populating does give them ability to change it in case they want to

**Comment by hariombalhara** [comment]
Yes but they already provided the response in Routing Form and allowing it to change store the data inconsistently.
Right now Routing Form stores its responses in a different table then Booking responses,so it's important to ensure that duplication is consistent.

**Comment by rksingh2001** [comment]
@hariombalhara Sorry, got busy in some personal stuff, will try to pick this up again on the weekend

**Comment by hariombalhara** [comment]
With this PR https://github.com/calcom/cal.com/pull/8780 routing form responses are automatically passed on to Booking Form. If there is a field by the same name/identifier, it would be automatically filled.

I am closing this ticket for now. If it makes sense to automatically hide the prefilled questions then we can open a separate enhancement ticket.
